junit-jupiter-api/junit-jupiter-api.gradle

```diff
@@ -2,8 +2,9 @@ description = 'JUnit Jupiter API'
 
 dependencies {
 	api("org.opentest4j:opentest4j:${ota4jVersion}")
-	api(project(":junit-platform-commons"))
-	compileOnly("org.jetbrains.kotlin:kotlin-stdlib")
+	api(project(':junit-platform-commons'))
+	compileOnly(project(':junit-platform-engine'))
```

Comment:
Let's ensure this shows up as an `optional` dependency in the generated Maven POM (see other comment in PR).